channels/bd.m3u

```diff
@@ -1,9 +1,61 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AnandTV.bd" tvg-country="BD" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/0IuoIXI.png" group-title="",Anand TV (720p)
-https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
-#EXTINF:-1 tvg-id="ChannelT1.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/dyoXd9c.png" group-title="",Channel T1 [Not 24/7]
+#EXTINF:-1 tvg-id="ADNBanglaUK.bd" tvg-country="UK" tvg-language="Bengali" tvg-logo="http://atnbanglauk.tv/wp-content/uploads/2015/04/sized1.png" group-title="General",ADN Bangla UK (576p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbanglauk-off.stream/playlist.m3u8
```

Comment:
Cannot connect. 

HTTP error 403 Forbidden

Reply:
403 on all https://cdn.appv.jagobd.com:444

Reply:
There are a bunch of :444 that do stream. only those listed above do not.

Reply:
They all work fine for me, so I guess some of them are just geo-blocked.